azure-devops/azure/devops/released/build/build_client.py

```diff
@@ -265,35 +265,21 @@ def get_builds(self, project, definitions=None, queues=None, build_number=None,
             query_parameters['repositoryType'] = self._serialize.query('repository_type', repository_type, 'str')
         response = self._send(http_method='GET',
                               location_id='0cd358e1-9217-4d94-8269-1c1ee6f93dcf',
-                              version='5.1',
+                              version='6.0',
                               route_values=route_values,
                               query_parameters=query_parameters)
-        response_value = self._deserialize('[Build]', self._unwrap_collection(response))
-        continuation_token = self._get_continuation_token(response)
-        return self.GetBuildsResponseValue(response_value, continuation_token)
-
-    class GetBuildsResponseValue(object):
-        def __init__(self, value, continuation_token):
-            """
-            Response for the get_builds method
-
-            :param value:
-            :type value: :class:`<[Build]> <azure.devops.v5_1.build.models.[Build]>`
-            :param continuation_token: The continuation token to be used to get the next page of results.
-            :type continuation_token: str
-            """
-            self.value = value
-            self.continuation_token = continuation_token
```

Comment:
It seems that here we are loosing support for continuation token, which was added in v5.1 in #258. Forgive me, if this is an intended change. I am not sure, how would I get continuation token while using v6.0